config/application.config.php

```diff
@@ -1,22 +1,29 @@
 <?php
-return [
-    // This should be an array of module namespaces used in the application.
-    'modules' => [
-        'ZendDeveloperTools',
-        'DoctrineModule',
-        'DoctrineORMModule',
-        'User',
-        'Application',
-        'Company',
-        'Decision',
-        'Frontpage',
-        'Activity',
-        'Photo',
-        'Education',
-        'Migration',
-        'MaglMarkdown'
-    ],
+$env = getenv('APP_ENV') ?: 'production';
```

Comment:
Well, the ZF2 tutorial made me sad.

-  It is a mess IMO
-  They use == instead of ===, so very weird things will happen if you set the environment to True.
- If you are using  ($env == 'production') three times, at least make an intermediate variable $isProduction = ($env === 'production'), which makes the code much more readable.